packages/core/src/App/Containers/Modals/deposit-now-or-later-modal/deposit-now-or-later-modal.scss

```diff
@@ -0,0 +1,18 @@
+.deposit-now-or-later-modal {
+    &.dc-dialog__wrapper {
+        z-index: 99999;
```

Comment:
Isn't it too big? 😅

Reply:
@niloofar-deriv Understand you, but in the components package we have some popups with `z-index: 9999`, I just wanted to be sure this modal will be above of all these components, because we need to see this modal above an other modal :) 

![image](https://github.com/binary-com/deriv-app/assets/120570511/77f8334c-622f-4582-8c5a-9fcfd4499062)

<img width="771" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/caf6818e-5e8a-4acf-a71d-c914091d61b9">

Reply:
OOPsss, ok then 😁